hooks/useHorizontalWheel.ts

```diff
@@ -0,0 +1,20 @@
+import { debounce } from 'lodash'
```

Comment:
```suggestion
import debounce from 'lodash/debounce'
```
lodash가 패키지 사이즈가 큰 편이라... tree shaking 관점에서도 import 방법을 한번 고민해봤으면 좋겠습니다요!!!

Reply:
- 사실 다른곳에서 사용하지 않으면 `setTimeout` 으로 만들어도 되긴하는데.. lodash 제거하고 고렇게 진행해볼까요!!
- 아 저렇게 속성으로 빼오게 되면 lodash 전체를 로드해오는 건가요? 처음 알았습니다..!

Reply:
- lodash는 범용성 높은 라이브러리니까 꼭 뺄 필요는 없는 것 같습니다 요것은 아론의 선택에 맡길게요👍
- lodash의 함정.. 디코에서도 말했지만 요 부분은 유명한 이슈라 함 확인해보시면 좋을 것 같네용

Reply:
- 넵 검색해보고 놀랬습니다,,😅
- lodash는 그럼 유지해두고 임포트만 변경하겠습니다!!